src/kirby/components/segmented-control/segmented-control.component.tns.html

```diff
@@ -0,0 +1,15 @@
+  <StackLayout orientation="horizontal" horizontalAlignment="left" class="segment-control">
+    <ng-container *ngFor="let item of segmentItems">
+      <GridLayout columns="auto, auto" (tap)="onSegmentTap(item)">
+        <ng-container *ngIf="item.badge">
+          <FlexBoxLayout class="segment-badge-wrapper" col="1" flexDirection="row" alignItems="center" justifyContent="flex-end">
+            <kirby-badge text="{{item.badge.content}}" colortype="{{item.badge.type}}"></kirby-badge>
+          </FlexBoxLayout>
+        </ng-container>
+        <StackLayout class="segment-item" col="0" colSpan="2" verticalAlignment="center" [ngClass]="{'is-checked': item.checked}">
```

Comment:
Consider substituting `[ngClass]="{'is-checked': item.checked}"` with `[div.is-checked]="item.checked"`

Reply:
I am not able to get that to work - also tried [StackLayout.is-checked].